extension-manifest-v3/src/background/telemetry/index.js

```diff
@@ -51,15 +51,23 @@ const loadStorage = async () => {
   return storage;
 };
 
-const getConf = async () => {
+const getConf = async (storage) => {
   const options = await store.resolve(Options);
 
+  if (!storage.installDate) {
+    storage.installDate =
+      options.installDate || new Date().toISOString().split('T')[0];
```

Comment:
Here (if it is not present in storage), it prefers it from options. Should the same be done with installRandomNumber? Or why is the options fallback needed? Also, how can the options be out-of-sync with the storage?